lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -109,14 +109,14 @@ confederation_id:
     config_set_append: '<state> confederation identifier <id>'
 
 confederation_peers:
-  # TODO: Supported in XR but broken as identifier is nvgened on a seperate line.
   default_value: ""
```

Comment:
From Glenn's readme update:

>Normally, if config_get_token produces no match, config_get() will return the defined default_value for this attribute. For some attributes, this may not be desirable. Setting auto_default: false will force config_get() to return nil in the non-matching case instead.

In this case, you want to config_get to return `default_confederation_peers` automatically, so you do not need to specify `auto_default: false`.